feat(main): style changes and mobile version

Araneline opened the pull request:
### 🔗 Linked issue

None

### ❓ Type of change

- [ ] 📖 Documentation (updates to the documentation or readme)
- [ ] 🐞 Bug fix (a non-breaking change that fixes an issue)
- [ ] 👌 Enhancement (improving an existing functionality like performance)
- [x] ✨ New feature (a non-breaking change that adds functionality)
- [ ] ⚠️ Breaking change (fix or feature that would cause existing functionality to change)

### 📚 Description

Hello Bellingcat team,
I've stumbled on your repo while browsing the open-source projects to contribute to and decided to have a look. Ended up spending my whole Sunday to make those changes :)

I've redesigned the initial project a little + added the initial styles for mobile view. Here's a video which shows what's ready by now: [new look](https://drive.google.com/file/d/1k7q00Ugug3bEhLsez4IQdW32E3n-SH_C/view)

My main aim was to make UI more **readable** and **intuitive**. 
Would be great to make a whole refactor of the project, but I decided against it for now, as mobile version would be something much more relevant in my opinion. Let me know if that's something you're interested in.

**That's not the final version of the changes I want to make (mobile UI is still in a horrible state), hence the draft and not a PR.**

Thank you for all your hard work!

### 📝 Checklist

- [ ] I have linked an issue or discussion.
- [ ] I have updated the documentation accordingly.

msramalho commented:
Hey @Araneline thanks for this!

Without diving into the code, these changes look good. I'm especially happy with the mobile look which would add a lot of value here. One remark is it should still have a reference to Forensic Architecture, maybe that can be near the leaflet/mapbox attribution:
![image](https://user-images.githubusercontent.com/19508417/196195534-f7c84297-113a-46ce-92de-f6311c41284c.png)

Other than that, optionally, if you can make the initial panel be close by clicking outside it (and not just the top right cross) that would also be a good UX change.

Let me know when you're ready for a code review.

Araneline commented:
@msramalho I've implemented an attribution part (figured out that top-left part of the screen is the best place for it). Added a bg to a popup with onclick => close popup. Reworked the timeline handles, made them available at any time. Also added some styles for better mobile experience. 

Please, ping me if there's any critical bug in the code, but I personally didn't find anything.

msramalho commented:
Hi @Araneline, again thanks for this contribution. The mobile support adds a lot of value. Breaking away from the original repo UI should be okay for now since most of the underlying logic was untouched. I'll release it soon.
